blind_walking/robots/a1.py

```diff
@@ -334,6 +335,23 @@ def GetFootForces(self):
 
     return contact_forces
 
+  def getFootDistToGround(self):
+    detect_dist_from_foot_to_ground = 5.0
+  
+    distances = [0] * NUM_LEGS
+    for foot_id in self._foot_link_ids:
+      toe_link_index = self._foot_link_ids.index(
+        foot_id)
+      foot_position = self._pybullet_client.getLinkState(self.quadruped, foot_id)[0] # index 0 is linkWorldPosition
+      projection_position = copy.copy(foot_position)
+      projection_position[2] = projection_position[2] - detect_dist_from_foot_to_ground # always pointing downwards
+      data = self._pybullet_client.rayTest(foot_position, projection_position)
+      if data[0] == -1: # -1 if did not detect anything within detectable distance (out of range)
+        distances[toe_link_index] = 5.0
```

Comment:
this looks like a bug, it should use `detect_dist_from_foot_to_ground` instead of being hardcoded

Reply:
Yes thats right! Thanks for checking!